sample/client/browser-sample.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { authManager } from "./authConstants";
+
+const subscriptionId = "00977cdb-163f-435f-9c32-39ec8ae61f4d";
+authManager.getToken().then((t) => {
```

Comment:
i have that in the final catch. Since all the calls are chained and I have a final catch in the end, if any promise fails, the control goes to the final catch.